### PR TITLE
[64_1] fix for de-embedding of images with unicode filename

### PR DESCRIPTION
**Pull request by jingkaimori**
## Why you open this Pull Request?

fix cork encoding of filename in "save image as..." dialog

## What work have you done in the current Pull Request?

- [x] fix

### CODE REVIEW

File: TeXmacs/progs/generic/embedded-edit.scm [hunk 1]
```diff
@@ -37,13 +37,13 @@
 
 (tm-define (embedded-suffix t)
```
Comment (darcy-shen) [inline TeXmacs/progs/generic/embedded-edit.scm:38]: please add unit tests for these two tm-defined interface

Comment (jingkaimori) [reply TeXmacs/progs/generic/embedded-edit.scm:38]: The second interface relies on `(current-directory)`, how to test it stably?

Comment (darcy-shen) [reply TeXmacs/progs/generic/embedded-edit.scm:38]: should use integration test but not unit test

Comment (jingkaimori) [reply TeXmacs/progs/generic/embedded-edit.scm:38]: unit test added @darcy-shen

**Review by darcy-shen — APPROVED** [review]
Verified locally via `xmake run 64_1`

**Pull request merged**